pkg/common/configs/config_test.go

```diff
@@ -239,12 +239,12 @@ func SerdeTest(t *testing.T, conf SchedulerConfig, description string) {
 }
 
 func CreateConfig(data string) (*SchedulerConfig, error) {
-	dir, err := ioutil.TempDir("", "test-scheduler-config")
+	dir, err := os.TempDir("", "test-scheduler-config")
```

Comment:
Does this even compile?

It looks like you can use either `os.TempDir()` (no args) or `os.MkdirTemp(dir, pattern string)`. Please check.

Reply:
sorry i made a mistake. change to : os.MkdirTemp("", "test-scheduler-config")